Client/Application/QuickActions.swift

```diff
@@ -46,8 +46,8 @@ struct QuickActions {
     static let TabURLKey = "url"
     static let TabTitleKey = "title"
 
-    private let lastBookmarkTitle = NSLocalizedString("Open Last Bookmark", comment: "String describing the action of opening the last added bookmark from the home screen Quick Actions via 3D Touch")
-    private let lastTabTitle = NSLocalizedString("Open Last Tab", comment: "String describing the action of opening the last tab sent to Firefox from the home screen Quick Actions via 3D Touch")
+    private let lastBookmarkTitle = NSLocalizedString("Open Last Bookmark", tableName: "3D Touch Actions", comment: "String describing the action of opening the last added bookmark from the home screen Quick Actions via 3D Touch")
```

Comment:
I don't know how well the l10n scripts handle table names with spaces in them. Because they translate to file names. If this turns out to be an issue after string freeze then we can handle it in l10n scripts probably.